src/app/loadout/plug-drawer/PlugSection.tsx

```diff
@@ -59,13 +66,21 @@ export default function PlugSection({
           header += ` (${headerSuffix})`;
         }
 
+        const key = header;
+
+        if (multiSelect) {
+          header += ` (${t(plugSet.overrideSelectedAndMax ?? tl('LB.SelectModsCount'), {
```

Comment:
I'd think the `tl` wouldn't be necessary here?

Reply:
`t(plugSet.overrideSelectedAndMax ?? 'LB.SelectModsCount', ...` ends up with i18next-scanner not extracting `SelectModsCount` to the english json.

Reply:
Ah, I see the reasoning now.